flow/layers/picture_layer.cc

```diff
@@ -26,9 +26,6 @@ void PictureLayer::Preroll(PrerollContext* context, const SkMatrix& matrix) {
 
     SkMatrix ctm = matrix;
     ctm.postTranslate(offset_.x(), offset_.y());
```

Comment:
So, this is a counter-example for getting rid of the ctm local (that doesn't mean we shouldn't get rid of it in other methods where it really is superficial).

Is there something different about the way that picture_layer handles the offset that might benefit the other uses of the raster cache?